src/simulation/openables/BirthdayPresent.ts

```diff
@@ -0,0 +1,11 @@
+import LootTable from '../../structures/LootTable';
+import SimpleOpenable from '../../structures/SimpleOpenable';
+
+const BirthdayPresentTable = new LootTable().oneIn(10, 'War ship').every('Slice of birthday cake');
```

Comment:
the wiki says this gives ever birthday item

additionally i dont think this makes much sense to have in general since its all guaranteed vanity items